lib/package_convenience.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+require_relative 'const'
+
+class PackageFunctions
+  def self.installed?(pkg)
```

Comment:
I'm thinking simply passing the package name would be better?  I only say this because currently it requires loading the package object prior to passing the parameter to the function and in this case, the object is not needed, only the package name.

Reply:
I'd prefer using the entire object for consistency, but in this case it does make more sense to just pass the package name.